Tests/XCUITests/JumpBackInTests.swift

```diff
@@ -0,0 +1,182 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+class JumpBackInTests: BaseTestCase {
+    func closeKeyboard() {
+        waitForExistence(app.buttons["urlBar-cancel"], timeout: TIMEOUT)
+        navigator.performAction(Action.CloseURLBarOpen)
+        navigator.nowAt(NewTabScreen)
+    }
+
+    func scrollDown() {
+        if !isTablet {
+            while app.staticTexts["Switch Your Default Browser"].exists || app.buttons["Learn How"].exists {
+                app.collectionViews["FxCollectionView"].swipeUp()
```

Comment:
I discovered that `swipe` on `FxCollectionView` works on M1. 🎉

Reply:
Yeah, The swipeUp used to work, the problem is the swipeLeft/swipeRight in some Views...